data/core/emptyview.lua

```diff
@@ -1,7 +1,11 @@
 local style = require "core.style"
 local keymap = require "core.keymap"
 local View = require "core.view"
+local command = require "core.command"
 
+-- This view is show in the background the first time lite xl is opened.
+-- Users also see it whenever they split the viewport and close all tabs
+-- in a single view.
```

Comment:
The second part of this comment is not true anymore in 2.0.4 and in `master`, right?